public_html/wp-content/mu-plugins/blocks/source/components/block-content/style.scss

```diff
@@ -1,4 +1,4 @@
-.wordcamp-block-content-none {
+.wordcamp-block_posts.has-no-content {
```

Comment:
`MUST` - Also `__`. There may be others too.

Reply:
Done in https://github.com/WordPress/wordcamp.org/pull/123/commits/92ab54753af3ccb3e8248f03065b2e340521a261